cloud/aws/lambda/monitors-lambda.tf

```diff
@@ -0,0 +1,121 @@
+resource "signalfx_detector" "pct_errors" {
```

Comment:
no heartbeat for lambda ?

Reply:
Based on the nature of lambda's I don't think it makes sense to do a heartbeat, but we can add back if you want